OpenRA.Mods.Common/UpdateRules/Rules/20190106/RenameSearchFromProcRadius.cs

```diff
@@ -0,0 +1,36 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2019 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class RenameSearchFromProcRadius : UpdateRule
+	{
+		public override string Name { get { return "SearchFromProcRadius renamed to SearchFromRefineryRadius"; } }
+		public override string Description
+		{
+			get
+			{
+				return "'Proc' is the RA abbreviation for [Ore] Processor.\n" +
+					"'Refinery' is the established generic name for where harvesters dump their resources.";
```

Comment:
Thinking out loud: "Harvester" and "Refinery" are still rather C&C-specific.  What if we did a larger scale rename to something like "ResourceCollector" and "ResourceUnloadDock"  to go alongside "StoresResources"?